.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
should not be used on markdown files

Reply:
maybe oriented here on [symfony editorconfig](https://github.com/symfony/symfony/blob/master/.editorconfig)

Reply:
added an exception for `.md` files. but i think you could also use a backslash instead of two spaces to insert a line break in markdown 🙂